pkg/admission/quota/clusterresourcequota/admission.go

```diff
@@ -102,8 +103,8 @@ func (q *clusterQuotaAdmission) Validate(ctx context.Context, a admission.Attrib
 		return nil
 	}
 
-	if !q.waitForSyncedStore(time.After(timeToWaitForCacheSync)) {
-		return admission.NewForbidden(a, errors.New("caches not synchronized"))
+	if !q.waitForSyncedStore(time.After(timeToWaitForCacheSync)) {		
+		return admission.NewForbidden(a, fmt.Errorf("%s-%s: caches not synchronized", accessor.GetNamespace(), accessor.GetName()))
```

Comment:
the name of the admission plugin is more useful.

`fmt.Errorf("quota.openshift.io/ClusterResourceQuota: caches not synchronized")`